unumpy/_multimethods.py

```diff
@@ -1421,3 +1565,209 @@ def _vander_default(x, N=None, increasing=False):
 @all_of_type(ndarray)
 def vander(x, N=None, increasing=False):
     return (x,)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def split(ary, indices_or_sections, axis=0):
+    return (ary,)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def array_split(ary, indices_or_sections, axis=0):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=2),
+)
+@all_of_type(ndarray)
+def dsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=1),
+)
+@all_of_type(ndarray)
+def hsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=0),
+)
+@all_of_type(ndarray)
+def vsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+def _tile_default(A, reps):
+    if ndim(reps) == 0:
+        reps = (reps,)
+
+    prep = len(reps) - ndim(A)
+    if prep > 0:
+        for _ in range(prep):
+            A = expand_dims(A, 0)
+    elif prep < 0:
+        reps = tuple(itertools.repeat(1, abs(prep))) + reps
+
+    for axis, dim in enumerate(reps):
+        if not isinstance(dim, numbers.Integral):
+            dim = dim[()]
+        dim = operator.index(dim)
+        arrays = [A] * dim
+        if len(arrays) == 0:
+            return asarray(arrays, dtype=A.dtype)
```

Comment:
Won't this give the wrong shape?

Reply:
You are right, I overlooked this. I'll use `reshape` here.